ministryofjustice/correspondence_tool_staff
# CT-2874 Update the settings feature flags to include QA environment

## Description
Update the settings file to set values for QA - these were omitted when the environment was created initially

## Self-review checklist


### Screenshots

![image](https://user-images.githubusercontent.com/1161161/84751215-3700e100-afb4-11ea-9d09-6b4bf00b4abe.png)


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2874

### Manual testing instructions

When you try to view an Offender SAR case on QA environment as Brian Rix, it would not allow you to even when everything else was correct because of the feature flag for offender SAR. You should be able to view offender SAR case as Offender SAR on QA provided everything else is correct.
